### PR TITLE
PA-117 edit apprentice details page

### PR DESCRIPTION
**Pull request by Pauljgraham**

### CODE REVIEW

File: src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Mappers/Apprentice/ConfirmEditApprenticeshipRequestToConfirmEditViewModelMapperTests.cs [hunk 1]
```diff
@@ -247,7 +262,7 @@ public class ConfirmEditApprenticeshipRequestToConfirmEditViewModelMapperTestsFi
     {
         private Mock<ICommitmentsApiClient> _mockCommitmentsApiClient;
 
-        public GetApprenticeshipResponse _apprenticeshipResponse;
+        public GetApprenticeshipResponse ApprenticeshipResponse;
```
[the remark this answers is not in the document]
Comment (Pauljgraham) [reply src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Mappers/Apprentice/ConfirmEditApprenticeshipRequestToConfirmEditViewModelMapperTests.cs:265]: it was originally a private which became a property (not by me), so I corrected it. But I'll add {get; set;} to make it explicit

[the remark this answers is not in the document]
Comment (benagain) [reply src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Mappers/Apprentice/ConfirmEditApprenticeshipRequestToConfirmEditViewModelMapperTests.cs:265]: It's not a property without {get; set;} - it's only a public field at the moment.  Turning it into a property is a good idea 👍

[the remark this answers is not in the document]
Comment (Pauljgraham) [reply src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Mappers/Apprentice/ConfirmEditApprenticeshipRequestToConfirmEditViewModelMapperTests.cs:265]: Actually made it an internal property

**Comment by benagain** [comment]
Merged as part of #312

**Pull request closed without merging**